flake.nix

```diff
@@ -0,0 +1,49 @@
+{
+  description = "# Zephyr™ Mechanical Keyboard (ZMK) Firmware";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.11";
+    flake-utils.url = "github:numtide/flake-utils";
+
+    # Customize the version of Zephyr used by the flake here
+    zephyr.url = "github:zephyrproject-rtos/zephyr/v3.2.0";
```

Comment:
Should this be the zmk fork instead? It can also be updated to 3.5 now too, https://github.com/zmkfirmware/zmk/pull/1995

Reply:
Yes, it should! Fixed.